### PR TITLE
Add OpenShift Connector Che Plugin v0.1.1

### PR DESCRIPTION
**Pull request by azatsarynnyy**
Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

### What does this PR do?
Adds OpenShift Connector Che Plugin v0.1.1 which is based on the VS Code extension v0.1.1.

The PR should be merged after https://github.com/eclipse/che-theia/pull/529 is merged and plugin image is published.

closes https://github.com/eclipse/che/issues/13406
closes https://github.com/eclipse/che/issues/13410

### CODE REVIEW

**Comment by amisevsk** [comment]
@azatsarynnyy Could you ping me when the image is published? I'll test it out.

**Comment by nickboldt** [comment]
If this is something we need to do in downstream CRW 2 too, @amisevsk can you submit a similar PR for https://github.com/redhat-developer/codeready-workspaces/tree/master/dependencies/che-plugin-registry ? I won't be around to review it but others should be able to (and I believe @dmytro-ndp  or @rhopp  can push it if you don't have push rights).

**Comment by azatsarynnyy** [comment]
@amisevsk sure, I'll ping you. Currently, waiting for a review of https://github.com/eclipse/che-theia/pull/529

**Comment by azatsarynnyy** [comment]
@amisevsk FYI: the image is published https://hub.docker.com/r/eclipse/che-remote-plugin-openshift-connector-0.1.1/tags

File: v3/plugins/redhat/vscode-openshift-connector/0.1.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-openshift-connector
+version: 0.1.1
+type: VS Code extension
+displayName: OpenShift Connector
+title: OpenShift Connector
+description: Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Eclipse Che
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins/redhat/vscode-openshift-connector
```
Comment (amisevsk) [inline v3/plugins/redhat/vscode-openshift-connector/0.1.1/meta.yaml]: Generally the repository should point at the repository that hosts the vscode plugin.

Comment (azatsarynnyy) [reply v3/plugins/redhat/vscode-openshift-connector/0.1.1/meta.yaml]: updated

**Pull request merged**